open-reac/src/main/java/com/powsybl/openreac/parameters/OpenReacAmplIOFiles.java

```diff
@@ -120,4 +120,10 @@ public boolean isDebug() {
     public AmplExportConfig getAmplExportConfig() {
         return amplExportConfig;
     }
+
+    @Override
+    public String getDebugDir() {
+        return "";
```

Comment:
If you don't want to support the debugDir parameter for now, you should return `null`.
(See [this line in powsybl-core](https://github.com/powsybl/powsybl-core/blob/9113f9a327d1e1cf9b627852f906d6c031c5e08f/computation-local/src/main/java/com/powsybl/computation/local/LocalComputationManager.java#L378). `getDebugDir()` is used to fill in `environment.getDumpDir()`)